conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/Registrable.java

```diff
@@ -23,6 +23,6 @@
  * opportunity to {@link #register} its (path, implementation handler) pairs with the given
  * {@link RoutingHandler Undertow router}.
```

Comment:
EndpointRegistry ConjureHandler

Reply:
?

Reply:
The javadoc comment references the wrong types